src/models/backend/safes.rs

```diff
@@ -0,0 +1,11 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct MasterCopy {
+    pub address: String,
+    pub version: String,
+    pub deployer: String,
```

Comment:
Do we need all of the attributes? 

If we add more attributes, the chances are higher that something fails because of changes to these attributes :P

Reply:
We don't this is just to match what the backend sends. I would then remove everything but `version` and `address`

Reply:
This is also used in the `about/master-copies` endpoint. Do you want us to hide fields from the forward to this endpoint?

Reply:
We would have to ask the clients what they required, but they might not require this endpoint at all anymore once this is merged